README.md

```diff
@@ -1042,7 +1022,7 @@ A k8s cluster consists of a set of worker machines, called nodes, that run conta
 
 The worker node(s) host the Pods that are the components of the application workload. The control plane manages the worker nodes and the Pods in the cluster. In production environments, the control plane usually runs across multiple computers, and a cluster usually runs multiple nodes, providing fault tolerance and high availability.
 
-- Control Plane Components
+#### Control Plane Components
```

Comment:
Another example of busy outline view:

<img width="291" alt="image" src="https://github.com/ByteByteGoHq/system-design-101/assets/50348/dc2e399b-b997-4a53-a0e7-b08b873043a1">